pkg/utils/host/workarounds.go

```diff
@@ -246,9 +246,9 @@ filesystemLoop:
 	for _, f := range fs {
 		var statfs unix.Statfs_t
 		for _, path := range f.paths {
-			err = unix.Statfs(path, &statfs)
-			if err != nil {
-				return mountsSuggested, fmt.Errorf("statfs %s: %w", path, err)
+			if err := unix.Statfs(path, &statfs); err != nil {
```

Comment:
nit: This indeed solves the issue, but I would prefer to avoid using named return values so that we don't end up with this shadowing.